chanzuckerberg/single-cell-data-portal
# feat(cellguide): 5207 SEO

## Reason for Change

- #5207
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

1. Update CellCard landing page to include meta tags for title and description
2. Update `Tissue` page to server side render page title, tissue name and description, and add SEO related meta tags
3. Update `CellType` page to server side render page title, name, meta tag for title and description
4. Add tests for all 3 pages to use snapshot tests to assert the correct meta tags are rendered
5. Fix flaky cellCard test steps
6. Fix hanging API requests that never get resolved

## Testing steps

1. The feature should work as normal
2. Inspect HTML page and you should now see the following:

<img width="1403" alt="Screenshot 2023-07-19 at 9 30 57 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/4eb03f6c-e4f2-47a0-a3c1-ce220741e32a">

## Notes for Reviewer
